Deeploy/Targets/Generic/Bindings.py

```
@@ -329,19 +330,35 @@
+        ConvTransposeTemplate.referenceTemplate1D,
+        BasicTransformer) for dtype in FloatDataTypes
+]
+
+BasicConvTranspose2DBindings = [
     NodeBinding(
         ConvChecker(
             [PointerClass(type), PointerClass(type), PointerClass(type)],  # input, weight, bias
             [PointerClass(type)]),
-        ConvTransposeTemplate.referenceTemplate,
+        ConvTransposeTemplate.referenceTemplate2D,
         BasicTransformer) for type in FloatDataTypes
```

Ruff (0.16.1)

[error] Variable `type` is shadowing a Python builtin

(A001)